ios/NewExpensify.xcodeproj/project.pbxproj

```diff
@@ -679,7 +683,7 @@
 				DEVELOPMENT_TEAM = 368M544MTT;
 				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
 				INFOPLIST_FILE = NewExpensify/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
Do you know if there's an equivalent iOS migration guide? Just in case jumping 2 versions leads to some deprecated code.

Reply:
There's a few like these [iOS 12](https://developer.apple.com/documentation/ios-ipados-release-notes/ios-12-release-notes), [iOS 13](https://developer.apple.com/documentation/ios-ipados-release-notes/ios-13-release-notes). However Xcode is more useful since it just highlights all deprecations for you:

<img src="https://user-images.githubusercontent.com/6833644/227268411-3433e9a1-e11a-43fe-8fe6-f2e030deaf42.png" width="300">

Most of these look to be linked to library code and were already around for a while (deprecated <= iOS 11). Since nothing's actually broken, I'd say we should probably :donothing: for these. If and when they break in a newer iOS release, we can just bump dependencies then.